### PR TITLE
Add `Search Examples` command to Next.js documentation extension

### PR DESCRIPTION
**Pull request by vimtor**
#1511 

## Description

- Add `Search Examples` command to search official Next.js examples: https://nextjs.org/examples
- Rename `Search Next.js Documentation` to `Search Documentation`
- Add `Next.js` as subtitle for all commands

## Screencast

https://user-images.githubusercontent.com/36263538/168490411-6872ee6c-2fef-4b12-9401-e92b173b1314.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/whatsapp/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Whatsapp Changelog
+
+## [Empty Screen] - 2022-03-26
+Add empty screen for the `Open Chat` command to avoid confusions
+
+## [Categories & Screenshots] - 2022-03-12
+Add categories and screenshots for the store
+
+## [Added Whatsapp] - 2021-12-06
+Initial version code
```
Comment (vimtor) [inline extensions/whatsapp/CHANGELOG.md:10]: @mattisssa you did something once with a README of my WhatsApp extension. Now it appears as a change each time I develop haha

Could we please fix it? I've tried everything but it always appears as a change.

Comment (vimtor) [reply extensions/whatsapp/CHANGELOG.md:10]: Here is the thing you did I mentioned: 
https://github.com/raycast/extensions/pull/1087/commits/396ecd6da375703b3ea1e1d3265eb0e27ff88a4b

Comment (mattisssa) [reply extensions/whatsapp/CHANGELOG.md:10]: Hey! Sorry, I didn't want to make an issue from my change 😬 
Have you rebased the change I made previously? All I did is just renamed the lowercased changelog file into uppercased. If you did the rebase I guess there still can be your old `changelog.md` file that can be removed now 🤔

Comment (pernielsentikaer) [reply extensions/whatsapp/CHANGELOG.md:10]: Do you need anything from us @vimtor

Comment (vimtor) [reply extensions/whatsapp/CHANGELOG.md:10]: I am not a 100% sure how to fix this. If you don't have a problem with this file being here I don't have more to say 😄

**Review by vimtor — COMMENTED** [review]
Remove Prasad as a contributor

File: extensions/whatsapp/CHANGELOG.md (hunk 1 above)
Comment (mattisssa) [reply extensions/whatsapp/CHANGELOG.md:10]: @vimtor let's leave it there then 😬

**Review by mattisssa — APPROVED** [review]
Hey! I've fixed a few small issues during the review. Sorry for taking the PR for so long, seems like it get lost for some reason.

**Pull request merged**